lms/djangoapps/commerce/views.py

```diff
@@ -0,0 +1,179 @@
+""" Commerce views. """
+
+import logging
+from simplejson import JSONDecodeError
+
+from django.conf import settings
+from django.utils.translation import ugettext as _
+import jwt
+from opaque_keys.edx.keys import CourseKey
+import requests
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.status import HTTP_406_NOT_ACCEPTABLE, HTTP_503_SERVICE_UNAVAILABLE, HTTP_202_ACCEPTED, HTTP_200_OK
```

Comment:
Minor: This line is long enough that it could benefit from being broken over multiple lines, as per [PEP 328](https://www.python.org/dev/peps/pep-0328/).